### PR TITLE
MNT: add arjanz as maintainer

### PR DESCRIPTION
**Pull request by MementoRC**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -55,4 +55,5 @@ about:
 
 extra:
   recipe-maintainers:
+    - arjanz
```
Comment (jakirkham) [inline recipe/meta.yaml:58]: @arjanz , could you please let us know if you want to be a maintainer here?

Comment (arjanz) [reply recipe/meta.yaml:58]: Hi @jakirkham, yes that seems like a logical choice as I'm the maintainer of the [py-ed25519-zebra-bindings](https://github.com/polkascan/py-ed25519-zebra-bindings) repos

Comment (MementoRC) [reply recipe/meta.yaml:58]: Thanks. @jakirkham the linter did not request Arjan's comment and I just merged, I should have understood that if it is required for staged recipe, it is required for feedstocks, I'll keep this in mind prior to merging next time

Comment (jakirkham) [reply recipe/meta.yaml:58]: Thank you both! 🙏

Yes it is best for maintainer updates to include confirmation from those affected by those changes

Comment (MementoRC) [reply recipe/meta.yaml:58]: Also, reading the doc more thoroughly, I would have seen that there is actually a process to follow for this purpose. Thank you

**Pull request merged**